packages/edit-site/src/components/sidebar/style.scss

```diff
@@ -15,4 +15,20 @@
 	.block-editor-block-inspector__card {
 		margin: 0;
 	}
+
```

Comment:
This is so the block styles don't leak into the sidebars (block inspector, global styles).

Reply:
Can we add these as comments?

Reply:
done at https://github.com/WordPress/gutenberg/pull/20530/commits/937578a5ada1fd9c05ad851d5ea27e3686bef66e